buddyboss/buddyboss-platform
# "Post Update" button is freezing when we post long URL in the activities.

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1113 .

### How to test the changes in this Pull Request:

1. Go to the Activity page
2. Post https://meetup.ubitto.com/playback/presentation/2.0/playback.html?meetingId=1f1b63dc873eed1ad256534923556b2a2e07c099-1592916832288
3. See error

### Proof Screenshots or Video
http://somup.com/cYiV2aQpM9


### Other information:
